package.json

```diff
@@ -60,6 +60,7 @@
     "styleguide:build": "vue-styleguidist --config styleguide.config.cjs build",
     "stylelint": "stylelint \"src/**/*.vue\" \"src/**/*.scss\" \"src/**/*.css\"",
     "stylelint:fix": "stylelint \"src/**/*.vue\" \"src/**/*.scss\" \"src/**/*.css\" --fix",
+    "check-types": "vue-tsc --noEmit",
```

Comment:
I seems that this script is usually called `typecheck` ot `type-check` (at least in Vute/Vue ecosystem). For example:
- [create-vue/template - `type-check`](https://github.com/vuejs/create-vue/blob/3498eb178d3f191b52593235dd0ca8691708c020/template/config/typescript/package.json#L5):
- [vite codebase - `typecheck`](https://github.com/vitejs/vite/blob/a0e1a0402648e0df60fb928ffd97b0230999990d/package.json#L25)

I'd go with `typecheck` to not intersect with existing package 👀

Reply:
Or `ts:check` (personal preference)

Reply:
In our libraries / apps:

- `type-check`
  - notify_push-client
  - nextcloud-moment
  - nextcloud-logger
  - nextcloud-browser-storage
- `test-types`
  - nextcloud-axios

Reply:
Should we can go with something consistent here, which can be later moved to workflows for all other repos?

Reply:
I would either stick with `check-types` as we do everywhere or prefer something like `lint:types` or `test:types` .
But I do not have a strong opinion on the naming and would accept any reasonable suggestion on how to proceed.

Reply:
> In our libraries / apps:

Updated :P

Reply:
`test` without a test runner seems wierd to me

Reply:
@ShGKme `spreed` uses `typescript:check` not `typecheck`: https://github.com/nextcloud/spreed/blob/d215b474519e6b386cde00df1a989ffd63712305/package.json#L12

Reply:
But yes as said I am fine with every name as long as it makes sense.
Currently I feel like `check-types` is the consistent name.
Alternatively `type-check` to follow Vite and Vue ecosystem.

> test without a test runner seems weird to me

well its a *test* that checks *types* so `test:types` run by tsc ;)
No but `test:...` also feels wrong, maybe `lint:types` but also weird. Probably best to either stick with one of the first mentioned.

Reply:
Asking the important questions I see :grin: 
I personally prefer `typecheck` or `typescript:check`

Reply:
Or `ts:check` to have fewer letters to type 🦥

Reply:
Renamed.